wiki/object-data-model.json

```diff
@@ -0,0 +1,34 @@
+//Customer
+{
+    "username": String, //Unique
+    "password": String, //Salted, Hashed Password
+    "fname": String,
+    "lname": String,
+    "email": String,
+    "address":  String
+}
+
+//Employee
+{
+    "username": String, //Unique
+    "password": String, //Salted, Hashed Password
+    "primary_task": ["DOUGH", "FROSTING", "DELIVERY", "CASHIER", "CLEANING"], //enum. primary task of the employee
```

Comment:
We might be able to consolidate tasks/add a task that is just "Other" or "Manager", just to also make sure we don't limit the tasks?